MobiusCore/Source/EffectHandlers/EffectRouter.swift

```diff
@@ -44,12 +46,15 @@ import Foundation
 ///     it has been started.
 public struct EffectRouter<Effect, Event> {
     private let routes: [Route<Effect, Event>]
+    fileprivate let metaData: EffectRouterMetaData
```

Comment:
Nit: [“metadata” is a noun](https://www.merriam-webster.com/dictionary/metadata) so maybe:
```suggestion
    fileprivate let metadata: EffectRouterMetadata
```